requirements/requirements-dev.txt

```diff
@@ -30,9 +30,7 @@ basicauth==0.4.1
     #   -r requirements/requirements.txt
```

Comment:
I think the reqs files shouldn't have any changes

Reply:
Am having the same problem as @castledan in #116, where running `pre-commit run --all-files` changes requirements. It is also based on local environment, tested with/without `venv` and get different results.

Reply:
Yes sorry this is totally my bad, you're right that it depends on your Python version, are you using 3.9? I'm using 3.8 but should probably upgrade, so you can keep these changes and we'll make sure it's explicit in Daniele's PR

Reply:
Yeah, I'm using 3.9.

Reply:
Thanks for clarifying, was about to manually remove but will leave as is!